pkg/image/builder.go

```diff
@@ -346,7 +351,7 @@ func copyOneImage(srcRegistry, destRegistry registry.RegistryOptions, image stri
 		}
 	}
 
-	if dryRun {
+	if !copyImages {
```

Comment:
would've been nicer to not have to pass this flag to this function since the function name is already `copyOneImage`, and it's a bit confusing that in some cases it doesn't _actually_ copy the image but only rewrite it. i understand that the previous logic in this function is needed here, but that could be moved to another common function i guess.

Reply:
renamed it....

Reply:
better :)